test/keys.js

```diff
@@ -23,6 +23,7 @@ describe('keys', function() {
   });
 
   it('works for primitives', function() {
+    /* jshint elision: true */
```

Comment:
(this also suggests we are not be linting test files at present -- 'cuz this does not get flagged by jshint without this change)